### PR TITLE
Update slurm examples to turn GVNIC on by default

### PR DESCRIPTION
**Pull request by alyssa-sm**
Update slurm examples to turn GVNIC on by default

### CODE REVIEW

File: examples/cae/cae-slurm.yaml [hunk 1]
```diff
@@ -208,7 +208,7 @@ deployment_groups:
       node_count_dynamic_max: 10
       machine_type: c3-highmem-176
       disk_type: 'pd-balanced'
-      bandwidth_tier: 'tier_1_enabled'
+      bandwidth_tier: gvnic_enabled
```
Comment (nick-stroud) [inline examples/cae/cae-slurm.yaml:211]: Having gvnic enabled is a pre-req to having tier 1 enabled. In addition you can think of tier 1 as an extra performance beyond gvnic. All that to say, places where we see tier 1 enabled, I would assume someone has already put some thought into it and I would leave it as tier 1.

File: examples/hpc-slurm.yaml
Comment (nick-stroud) [inline examples/hpc-slurm.yaml:1]: I think you are missing all the examples in `community/examples/*`.

**Review by nick-stroud — CHANGES_REQUESTED** [review]

File: examples/cae/cae-slurm.yaml (hunk 1 above)
Comment (alyssa-sm) [reply examples/cae/cae-slurm.yaml]: Makes sense. I changed the ones originally marked as tier 1 back.

File: examples/hpc-slurm.yaml
Comment (alyssa-sm) [reply examples/hpc-slurm.yaml:1]: Updated!

File: examples/hpc-enterprise-slurm.yaml [hunk 2]
```diff
@@ -142,7 +142,7 @@ deployment_groups:
         family: $(vars.family)
         project: $(vars.project)
       instance_image_custom: true
-      bandwidth_tier: tier_1_enabled
+      bandwidth_tier: 'tier_1_enabled'
```
Comment (nick-stroud) [inline examples/hpc-enterprise-slurm.yaml:145]: Remove the quotes and just leave it as it was.

File: examples/image-builder.yaml [hunk 3]
```diff
@@ -75,6 +75,7 @@ deployment_groups:
       instance_image:
         family: $(vars.new_image_family)
         project: $(vars.project_id)
+      bandwidth_tier: gvnic_enabled
```
Comment (nick-stroud) [inline examples/image-builder.yaml:78]: nit: I would put this below the `instance_image_custom` setting to keep the image settings next to each other.

File: community/examples/intel/hpc-intel-select-slurm.yaml [hunk 4]
```diff
@@ -130,6 +138,8 @@ deployment_groups:
         project: $(vars.project_id)
       max_node_count: 100
       machine_type: c2-standard-60
+      bandwidth_tier: 'gvnic_enabled'
```
Comment (nick-stroud) [inline community/examples/intel/hpc-intel-select-slurm.yaml:141]: nit: remove quotes

File: community/examples/AMD/hpc-amd-slurm.yaml
Comment (nick-stroud) [inline community/examples/AMD/hpc-amd-slurm.yaml:1]: Please make sure to run tests where affected.

**Review by nick-stroud — APPROVED** [review]

File: examples/hpc-enterprise-slurm.yaml (hunk 2 above)
Comment (alyssa-sm) [reply examples/hpc-enterprise-slurm.yaml]: Done.

File: examples/image-builder.yaml (hunk 3 above)
Comment (alyssa-sm) [reply examples/image-builder.yaml]: Done.

File: community/examples/intel/hpc-intel-select-slurm.yaml (hunk 4 above)
Comment (alyssa-sm) [reply community/examples/intel/hpc-intel-select-slurm.yaml]: Done.

**Pull request merged**